code/extensions/FluentFilteredExtension.php

```diff
@@ -113,6 +106,7 @@ public function updateCMSFields(FieldList $fields)
      */
     public function augmentDataQueryCreation(SQLQuery $query, DataQuery $dataQuery)
     {
+        $dataQuery->setQueryParam('Fluent.FilterAdmin', Fluent::config()->FilterAdmin);
```

Comment:
Can you please document this in the docs, and make sure the config option follows the naming convention (`filter_admin`).

You'll need to make sure there's a default option set, which should be false, right? Add to https://github.com/tractorcow/silverstripe-fluent/blob/master/_config/fluent.yml